package.json

```diff
@@ -1,37 +1,35 @@
 {
   "name": "test",
   "scripts": {
-    "build": "waffle",
-    "test": "export NODE_ENV=test && mocha -r ts-node/register 'test/**/*.ts'",
-    "btest": "yarn run build && yarn run test",
```

Comment:
If I run `yarn test` without `yarn build` the contracts are compiled anyway, but the tests fail with a TS error.

Reply:
Yes, this is expected. This is because tests are run with `buidler`, but they are Waffle tests with raw ethersjs. I am working on coverting them to 100% buidler tests but that is a separate PR.